core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -1196,15 +1148,16 @@ public static boolean isValidTablePropertyKey(String key) {
    */
   public static final EnumSet<Property> HOT_PATH_PROPERTIES = EnumSet.of(
       Property.TSERV_CLIENT_TIMEOUT, Property.TSERV_TOTAL_MUTATION_QUEUE_MAX,
-      Property.TSERV_ARCHIVE_WALOGS, Property.GC_TRASH_IGNORE, Property.TSERV_MAJC_DELAY,
-      Property.TABLE_MINC_LOGS_MAX, Property.TSERV_MAJC_MAXCONCURRENT,
-      Property.REPLICATION_WORKER_THREADS, Property.TABLE_DURABILITY, Property.INSTANCE_ZK_TIMEOUT,
-      Property.TABLE_CLASSPATH, Property.MASTER_METADATA_SUSPENDABLE,
-      Property.TABLE_FAILURES_IGNORE, Property.TABLE_SCAN_MAXMEM);
+      Property.GC_TRASH_IGNORE, Property.TSERV_MAJC_DELAY, Property.TABLE_MINC_LOGS_MAX,
+      Property.TSERV_MAJC_MAXCONCURRENT, Property.REPLICATION_WORKER_THREADS,
+      Property.TABLE_DURABILITY, Property.INSTANCE_ZK_TIMEOUT, Property.TABLE_CLASSPATH,
+      Property.MASTER_METADATA_SUSPENDABLE, Property.TABLE_FAILURES_IGNORE,
+      Property.TABLE_SCAN_MAXMEM);
 
-  private static final EnumSet<Property> fixedProperties = EnumSet.of(Property.TSERV_CLIENTPORT,
-      Property.TSERV_NATIVEMAP_ENABLED, Property.TSERV_SCAN_MAX_OPENFILES,
-      Property.MASTER_CLIENTPORT, Property.GC_PORT);
+  private static final EnumSet<Property> fixedProperties = EnumSet.of(
+      Property.TSERV_SCAN_MAX_OPENFILES, Property.TSERV_CLIENTPORT,
+      Property.TSERV_NATIVEMAP_ENABLED, Property.TSERV_SCAN_MAX_OPENFILES, Property.GC_PORT,
+      Property.MASTER_CLIENTPORT);
```

Comment:
It's not clear why `TSERV_SCAN_MAX_OPENFILES` was added a second time to this set, or why `MASTER_CLIENTPORT` was added.

Reply:
`TSERV_SCAN_MAX_OPENFILES` should not have been but `MASTER_CLIENTPORT` was not added in this work (the order was changed from before, i have reverted the reordering in the next commit)